docs/stack/subgraph/queries.md

```diff
@@ -0,0 +1,346 @@
+You can use GraphQL queries to get quick info from [DAOstack Sunbgraph](https://thegraph.com/explorer/subgraph/daostack/v30_1) hosted on GraphExplorer.
```

Comment:
Latest now is https://thegraph.com/explorer/subgraph/daostack/v31_0 but you might want to link here to master (https://thegraph.com/explorer/subgraph/daostack/master), we don't use it in production but it's updated to the latest master branch on the repo itself.